ci-operator/jobs/openshift/cloud-credential-operator/openshift-cloud-credential-operator-master-periodics.yaml

```diff
@@ -0,0 +1,78 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 0 * * 0
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: cloud-credential-operator
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc-cron
+  reporter_config:
+    slack:
+      channel: '#team-hive-alert'
+      job_states_to_report:
+      - failure
+      - error
+      report_template: Job {{ .Spec.Job }} failed. {{ .Status.URL|View logs }}
```

Comment:
@2uasimojo I want to tag hive-cop here but I either don't know how to escape `@` in yaml or `make update` doesn't like it. Any suggestions?